src/App.tsx

```diff
@@ -2,72 +1,70 @@
+import React, { useMemo, useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
+
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+
+import { SearchInput } from './components/searchInput/searchInput';
+import Content from './components/content/content';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [delay] = useState(300); // for change add setDelay
+
+  const [fullField, setFullField] = useState('');
+  const [selectedPerson, setSelectedPerson] = useState<Person | ''>('');
```

Comment:
The `selectedPerson` state is initialized with an empty string, which might lead to type inconsistencies since `selectedPerson` is expected to be of type `Person`. Consider initializing it with `null` or `undefined` if you want to represent a non-selected state, and adjust the type accordingly.